src/components/App/App.css

```diff
@@ -180,11 +180,12 @@ textarea {
   padding-left: 40px;
   padding-right: 40px;
   padding-bottom: 40px;
-  display: inline-block;
+  /* display: inline-block; */
```

Comment:
Can unused css just be deleted?

Reply:
Whoops!  I for sure need to be more confident about deleting stuff in instead of just commenting it out.